main/src/library/Int32.flix

```diff
@@ -242,32 +242,32 @@ namespace Int32 {
     /// The bits of x have positions: 0 (rightmost bit) - 31 (leftmost bit)
     ///
     @Time(1) @Space(1)
-    pub def getBit(x: Int32, position: Int32): Int32 =
-        if ((x >>> position) rem 2 == 0) 0 else 1
+    pub def getBit(position: {position :: Int32}, x: Int32): Int32 =
```

Comment:
Since both arguments are `Int32` i made `position` a record. Do we want the whole word or something like `pos`?

Reply:
Yes, lets do it!